inc/addons/breadcrumbs/customizer/class-astra-breadcrumbs-color-configs.php

```diff
@@ -43,21 +43,28 @@ public function register_configuration( $configurations, $wp_customize ) {
 
 			$defaults = Astra_Theme_Options::defaults();
 
+			if ( Astra_Builder_Helper::$is_header_footer_builder_active ) {
```

Comment:
```suggestion
			if ( true === Astra_Builder_Helper::$is_header_footer_builder_active ) {
```

Please perform explicit checks everywhere possible.

Reply:
Added improvement in the same task - Task for this - https://trello.com/c/8NHUTOuS/1191-customizer-redesign-pr-review-changes